pkgs/top-level/ocaml-packages.nix

```diff
@@ -1232,5 +1232,5 @@ in let inherit (pkgs) callPackage; in rec
 
   ocamlPackages_latest = ocamlPackages_4_10;
 
-  ocamlPackages = ocamlPackages_4_08;
+  ocamlPackages = ocamlPackages_4_09;
```

Comment:
What are our policies for keeping version of the ocaml compiler though?
Which of these version are still maintained upstream?